src/radar-chart.js

```diff
@@ -372,3 +372,5 @@ var RadarChart = {
       .call(chart);
   }
 };
+
+module.exports = RadarChart;
```

Comment:
Maybe that will be nice to check if module exists? Someone can use it in as a bower package.